hadoop-ozone/ozonefs-common/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneClientUtils.java

```diff
@@ -72,6 +76,34 @@ public void testEmptyKeyName() throws IOException {
     assertNull(checksum);
   }
 
+  @Test
+  public void testGetFileChecksumForECFileShouldReturnNull()
+      throws IOException {
+    OzoneVolume volume = mock(OzoneVolume.class);
+    OzoneBucket bucket = mock(OzoneBucket.class);
+    ClientProtocol clientProtocol = mock(ClientProtocol.class);
+    OzoneManagerProtocol ozoneManagerProtocol =
+        mock(OzoneManagerProtocol.class);
+    try {
+      OmKeyInfo omKeyInfo = new OmKeyInfo.Builder()
+          .setReplicationConfig(new ECReplicationConfig("rs-3-2-1024K"))
+          .setVolumeName("vol").setBucketName("bucket").setKeyName("mykey")
+          .setDataSize(10).setCreationTime(1).setModificationTime(1).build();
+      Mockito.when(clientProtocol.getOzoneManagerClient())
+          .thenReturn(ozoneManagerProtocol);
+      Mockito.when(ozoneManagerProtocol.lookupKey(any())).thenReturn(omKeyInfo);
+      FileChecksum checksum = OzoneClientUtils
+          .getFileChecksumWithCombineMode(volume, bucket, "test", 1,
+              OzoneClientConfig.ChecksumCombineMode.MD5MD5CRC, clientProtocol);
+      assertNull(checksum);
+    } finally {
+      Mockito.reset(volume);
+      Mockito.reset(bucket);
+      Mockito.reset(volume);
+      Mockito.reset(volume);
```

Comment:
i'm not familiar with this. Is it required to reset volume multiple times?

Reply:
Oops, this is my mistake. I meant to clean all other mocks :-)